XenAdmin/Wizards/GenericPages/SelectVMStorageWithMultipleVirtualDisksPage.cs

```diff
@@ -409,11 +430,17 @@ private bool CheckStorageRequirements(out string error)
 
 					var storageDetail = (StorageDetail)row.Cells[0].Tag;
 					string uuid = selectedItem.item.uuid;
+					ulong requiredSpace = storageDetail.RequiredSpace;
+
+					if (!IsExtraSpaceNeeded(storageDetail.ResourceData.SR, XenRef<SR>.Create((selectedItem.item.opaque_ref))))

```

Comment:
This can be written as: 
```
var requiredSpace = IsExtraSpaceNeeded(storageDetail.ResourceData.SR, new XenRef<SR>(toStringWrapper.item))
    ? storageDetail.RequiredSpace : 
    0;
```

Reply:
Nice!